src/main/java/carriermanagementsystem/CarrierManagementSystemDirector.java

```diff
@@ -19,48 +19,47 @@
  * carrier it creates.
  */
 public class CarrierManagementSystemDirector implements CarrierManagementIf {
-    private static volatile CarrierManagementSystemDirector UNIQUE_INSTANCE = new CarrierManagementSystemDirector();
-    private final int LTE_MAXIMUM_CARRIER_ID = 500; 
+    private static final CarrierManagementSystemDirector UNIQUE_INSTANCE = new CarrierManagementSystemDirector();
+    private final int LTE_MAXIMUM_CARRIER_ID = 500;
     private final int WCDMA_MAXIMUM_CARRIER_ID = 500;
     private Stack<Integer> lteCarrierIdGenerator = new Stack<>();
     private Stack<Integer> wcdmaCarrierIdGenerator = new Stack<>();
 
-	// For the future usage.
-	// This list will save all carrier object and carrierID so the object can be
-	// modified from radio unit.
-	// private ConcurrentHashMap<Integer,Carrier> carrierList;
+    // For the future usage.
```

Comment:
Since RadioUnits will have a list of Carriers, this comment here can probably be removed entirely.

Reply:
Done